tools/main.py

```diff
@@ -72,8 +74,9 @@ def main():
 
     print('Building main model: {}'.format(cfg.model.name))
     model = torchreid.models.build_model(**model_kwargs(cfg, num_train_classes))
-    num_params, flops = compute_model_complexity(model, (1, 3, cfg.data.height, cfg.data.width))
-    print('Main model complexity: params={:,} flops={:,}'.format(num_params, flops))
+    macs, num_params = get_model_complexity_info(model, (3, cfg.data.height, cfg.data.width),
+                                                 as_strings=False, verbose=False, print_per_layer_stat=False)
+    print('Main model complexity: params={:,} flops={:,}'.format(num_params, macs * 2))
```

Comment:
Have you checked the transformer model complexity computing?

Reply:
It's still wrong, but now the numbers are above than the expected ones: M params=24.946112 G flops=22.183603248 for V3_large_21k

Reply:
seems like now it's okay